ansible/roles/osdsdock/tasks/main.yml

```diff
@@ -13,6 +13,14 @@
 # limitations under the License.
 
 ---
+- name: install tgt and ceph-common external packages
+  apt:
+    name: "{{ item }}"
+    state: present
+  with_items:
+    - tgt
+    - ceph-common
```

Comment:
I see here that you installed ceph-common.  I guess that's why you removed the libs earlier.

Reply:
You can take a look at `Makefile` under opensds repo.

Reply:
Sure, you can clean the environment up and just run `ansible-playbook site.yml -i localhost` under `opensds-installer/ansible` folder without `sudo` access.

Reply:
Got it. Thanks.